validation/config_updates_without_affect.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	tap "github.com/mndrix/tap-go"
+	"github.com/mrunalp/fileutils"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-tools/generate"
+	"github.com/opencontainers/runtime-tools/specerror"
+	"github.com/opencontainers/runtime-tools/validation/util"
+	uuid "github.com/satori/go.uuid"
+)
+
+func main() {
+	t := tap.New()
+	t.Header(0)
+
+	bundleDir, err := util.PrepareBundle()
+	if err != nil {
+		util.Fatal(err)
+	}
+	defer os.RemoveAll(bundleDir)
+
+	r, err := util.NewRuntime(util.RuntimeCommand, bundleDir)
+	if err != nil {
+		util.Fatal(err)
+	}
+
+	testPath := filepath.Join("/", "test", "config.json")
```

Comment:
This rooted path fails when you use it for `g.SaveToFile(…)` below:

```console
$ ./validation/config_updates_without_affect.t 
TAP version 13
open /test/config.json: no such file or directory
```

I agree with @liangchenye that [you should set this path to something inside the bundle][1], at least for the host-mount-namespace `SaveToFile` call.

[1]: https://github.com/opencontainers/runtime-tools/pull/604#discussion_r175666390